tools/clang/test/CodeGenSPIRV/vk.binding.register.hlsl

```diff
@@ -61,6 +61,20 @@ ConstantBuffer<S> myCbuffer3 : register(b2, space3);
 // CHECK-NEXT: OpDecorate %sbuffer2 Binding 6
 RWStructuredBuffer<S> sbuffer2 : register(u6, space1);
 
+    // The counter variable will use the next unassigned number
+// CHECK:      OpDecorate %abuffer DescriptorSet 0
+// CHECK-NEXT: OpDecorate %abuffer Binding 5
```

Comment:
I was under the impression that the (descriptorSet, Binding) pair should uniquely identify the resource we're referring to.
I see (0,5) used for both `abuffer` and `sbuffer1`. Am I missing something?

Reply:
Yes, we should have different pairs for different resources. However, the current implementation doesn't differentiate `t`, `s`, `u`, `b` yet. `sbuffer1` is bound to `t5`, while `abuffer` is bound to `u5`.  But we only take the number into consideration so they are all mapped to binding number 5. We need a way to automatically assign non overlapping numbers eventually.

Reply:
Ok, makes sense